lib/saml_idp_constants.rb

```diff
@@ -37,11 +43,28 @@ module Constants
 
       REQUESTED_ATTRIBUTES_CLASSREF = 'http://idmanagement.gov/ns/requested_attributes?ReqAttr='
 
-      VALID_AUTHN_CONTEXTS = IdentityConfig.store.valid_authn_contexts.freeze
-      IAL2_AUTHN_CONTEXTS = [IAL2_AUTHN_CONTEXT_CLASSREF, LOA3_AUTHN_CONTEXT_CLASSREF].freeze
-      BIOMETRIC_IAL_CONTEXTS = [IAL2_BIO_PREFERRED_AUTHN_CONTEXT_CLASSREF,
-                                IAL2_BIO_REQUIRED_AUTHN_CONTEXT_CLASSREF].freeze
+      VALID_AUTHN_CONTEXTS = (if FeatureManagement.use_semantic_authn_contexts?
+                                IdentityConfig.store.valid_authn_contexts_semantic
+                             else
+                               IdentityConfig.store.valid_authn_contexts
+                             end).freeze
 
+      BIOMETRIC_IAL_CONTEXTS = [
+        IAL_VERIFIED_FACIAL_MATCH_REQUIRED_ACR,
+        IAL_VERIFIED_FACIAL_MATCH_PREFERRED_ACR,
+        IAL2_BIO_REQUIRED_AUTHN_CONTEXT_CLASSREF,
+        IAL2_BIO_PREFERRED_AUTHN_CONTEXT_CLASSREF,
+      ].freeze
+      BIOMETRIC_REQUIRED_IAL_CONTEXTS = [
```

Comment:
[nitpick] could you add a newline inbetween the multiline arrays for readability? thank you!